libs/platform/ebpf_platform.h

```diff
@@ -66,6 +66,20 @@ extern "C"
         size_t length;
     } ebpf_utf8_string_t;
 
+    typedef enum _ebpf_pool_tag
+    {
+        EBPF_POOL_TAG_DEFAULT = 'fpbe',
+        EBPF_POOL_TAG_PROGRAM = 'grpe',
+        EBPF_POOL_TAG_CORE = 'roce',
+        EBPF_POOL_TAG_MAP = 'pame',
+        EBPF_POOL_TAG_LINK = 'knle',
+        EBPF_POOL_TAG_EPOCH = 'cpee',
+        EBPF_POOL_TAG_STATE = 'atse',
+        EBPF_POOL_TAG_ASYNC = 'nsae',
+        EBPF_POOL_TAG_NATIVE = 'vtne',
+        EBPF_POOL_TAG_RING_BUFFER = 'fbre',
```

Comment:
The order currently looks random.  Sort in alphabetical order

Reply:
Maybe instead of pool tags we could embed some better location information in the memory itself?

Proposal:
Reserve 1 QWORD at the start of the memory block and encode calling address into it. See ebpf_epoch_allocate for an example of embedding a prefix onto a block. We can keep a single pool tag, but get exact allocation address.

From the debugger we can then do:
dqs <address> 1 and get the allocated address.

Reply:
Agree. Another option I was thinking was to use numbers starting with 1 for tags and create more fine-grained pool tags instead of just `EBPF_POOL_TAG_PROGRAM`, `EBPF_POOL_TAG_CORE`, etc.